### ISSUE TITLE
[DOCUMENT]: GIBCT Section 116 - Excel document uses color as the only means to convey intent

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#3756]
## Description

VA 508 office has identified the [Excel Download Data on All Schools](https://www.benefits.va.gov/GIBILL/docs/job_aids/ComparisonToolData.xlsx) has some areas that use color as the only way to convey intent. This is a violation of Success Criteria 1.4.1. Screenshot attached below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

* Color is complemented by text, icons, or other means to convey intent

## WCAG or Vendor Guidance (optional)

* [Use of Color: Understanding SC 1.4.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-without-color.html)

## Screenshots or Trace Logs

![Screen Shot 2019-11-26 at 3 48 48 PM](https://user-images.githubusercontent.com/934879/69676464-6d9dfb00-1066-11ea-892e-53c6c9d742c8.png)

**Comment by desireeturner** [comment]
@1Copenut The maintenance of this file is managed by the EDU team, so we will make recommendations to them regarding this update.

**Comment by desireeturner** [comment]
@1Copenut The Education team has been informed of this issue.

**Comment by desireeturner** [comment]
@1Copenut The file has been updated by EDU, so I believe this is ready for a round of testing.

**Comment by 1Copenut** [comment]
@desireeturner Thank you! I've let the VA 508 office know this Excel doc is ready for retesting.

**Comment by 1Copenut** [comment]
@desireeturner This issue appears to be resolved. I'll move it to close.
